Makefile

```diff
@@ -33,11 +45,14 @@ cleanrun: clean run
 # Install requirements
 install:
 	python -m pip install --upgrade pip
-	pip install -r requirements.txt -r requirements-dev.txt
+	pip install -r requirements.txt
+
+install-dev: install
+	pip -r requirements-dev.txt
 
-# Setup virtuel environment
+# Setup virtual environment
 venv:
 	python -m venv .pyenv
-	.pyenv\Scripts\activate && make install	
+	source .pyenv/bin/activate && make install
```

Comment:
Does not work on Windows

Reply:
Can we support Windows and UNIX systems at the same time? There are multiple activate-scripts even for different shells...

Reply:
This might help with that.
https://stackoverflow.com/questions/4058840/makefile-that-distincts-between-windows-and-unix-like-systems